go.mod

```diff
@@ -4,9 +4,13 @@ go 1.16
 
 require (
 	github.com/aws/aws-sdk-go v1.38.40
+	github.com/aws/aws-sdk-go-v2 v1.16.2
+	github.com/aws/aws-sdk-go-v2/config v1.15.3
+	github.com/aws/aws-sdk-go-v2/service/ec2 v1.34.0
```

Comment:
What are the compatibility rules between these modules? They seem related but are clearly versioned separately.

Reply:
each submodule has different version calculation. u can check the v1.16.2 release.